src/common/snippets/src/op/serialize_node.cpp

```diff
@@ -28,22 +28,22 @@ void SerializationNode::validate_and_infer_types() {
 
 std::shared_ptr<Node> SerializationNode::clone_with_new_inputs(const OutputVector &new_args) const {
     check_new_args_count(this, new_args);
-    return std::make_shared<SerializationNode>(new_args.at(0), m_expr);
+    return std::make_shared<SerializationNode>(new_args, m_expr);
 }
 
 bool SerializationNode::visit_attributes(AttributeVisitor &visitor) {
     std::vector<std::pair<std::string, ov::PartialShape>> shapes;
-    const auto &node = m_expr->get_node();
-    for (size_t i = 0; i < node->get_input_size(); i++) {
-        const auto &pshape = node->get_input_partial_shape(i);
-            if (pshape.begin() != pshape.end())
-                shapes.emplace_back("in_shape_" + std::to_string(i), node->get_input_partial_shape(i));
+    for (size_t i = 0; i < m_expr->get_input_count(); i++) {
+        const auto &shape = m_expr->get_input_port_descriptor(i)->get_shape();
+        if (!shape.empty())
+            shapes.emplace_back("in_shape_" + std::to_string(i), ov::PartialShape(shape));
```

Comment:
Just simple question since I'm confused a little bit: Could you tell me please if there are some limitations that we have to convert `std::vector<size_t>` to `ov::PartialShape`? 😃

Reply:
Actually this was done by mistake, no limitations exist) I removed conversion to `ov::PartialShape`